src/richie/apps/courses/templates/courses/cms/fragment_course_glimpse.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n cms_tags %}
+{% comment %}Obviously, context template variable "page" and "course" are required and must be a CMS page instance{% endcomment %}
+{% with container=container_name|default:"course-card" glimpse_info=course.glimpse_info %}
```

Comment:
If we don't have another use case yet, I think we should assume that there is only 1 course glimpse and hard code the class name. 

I we need a different course glimpse one day, it will most likely be cleaner to add it in a separate fragment.